src/controls/dimension-slider.js

```diff
@@ -517,7 +517,13 @@ export default class DimensionSlider {
      */
     getZProjectionDisabled(handle, forwards) {
         let dims = this.image_config.image_info.dimensions;
-        return (handle !== null && forwards || (dims.max_x * dims.max_y) > UNTILED_RETRIEVAL_LIMIT);
+        let tiled = (dims.max_x * dims.max_y) > UNTILED_RETRIEVAL_LIMIT;
+        let bytes_per_pixel = Math.ceil(Math.log2(this.image_config.image_info.range[1]) / 8.0);
```

Comment:
you may want to add a ``TODO`` and create an issue so we can use the ``omero-model`` property when available https://github.com/ome/omero-model/pull/47

Reply:
Ah interesting, so: see if someone set a value explicitly for iviewer, then check the server default, then use a client fallback.

Reply:
Currently we don't have the config available from the server.
When that is available, we could teach iviewer to use that as a default value, but if it is set in iviewer itself then use that. That could allow iviewer to be more restrictive than the server (but not *less* restrictive since the server would enforce it's own limit).